script.js

```diff
@@ -1,4 +1,100 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+let numOfDice = 0;
+let gameMode = "waiting for number of players";
+let playerTurn = 0;
+let arrayOfPlayers = [];
```

Comment:
Good job using these global variables to track multiple things,
remember your SCREAM_CASE constants, it will be useful in not only preventing typos but identifying where in your code these appear (i.e when modes change)